installer/deb/create_deb.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+# This script generates a .deb file for the installation of MassaStation on a Debian-based Linux distribution.
+
+BUILD_DIR=builddeb
+PKGVERSION=0.0.0-dev
+
+DEB_NAME=massastation_$PKGVERSION\_amd64.deb
+SERVER_BINARY_NAME=massastation-server
+APP_BINARY_NAME=massastation-app
+
+# Print error message to stderr and exit with code 1.
+fatal() {
+    echo "FATAL: $1"
+    exit 1
```

Comment:
I'm not sure this will behave as you're expeting:
as you are in a function call this will exit the function fatal, not stop the script execution.

You can fix that by using the `set -e` command.